docassemble/ALWeaver/interview_generator.py

```diff
@@ -243,20 +245,20 @@ def fill_in_docx_attributes(
         else:
             self.field_type_guess = "text"
 
-    def fill_in_pdf_attributes(self, pdf_field_tuple):
+    def fill_in_pdf_attributes(self, pdf_field_tuple: Any) -> None:
         """Let's guess the type of each field from the name / info from PDF"""
         # The raw name of the field from the PDF: must go in attachment block
-        self.raw_field_names: List[str] = [pdf_field_tuple[0]]
+        self.raw_field_names = [pdf_field_tuple[0]]
         # turns field_name into a valid python identifier: must be one per field
-        self.variable: str = remove_multiple_appearance_indicator(
+        self.variable = remove_multiple_appearance_indicator(
             varname(self.raw_field_names[0])
         )
         # the variable, in python: i.e., users[1].name.first
-        self.final_display_var: str = map_raw_to_final_display(self.variable)
+        self.final_display_var = map_raw_to_final_display(self.variable)
 
         variable_name_guess = self.variable.replace("_", " ").capitalize()
         self.has_label = True
-        self.maxlength = get_character_limit(pdf_field_tuple)
+        self.maxlength = get_character_limit(pdf_field_tuple) or 80
```

Comment:
Only question--do we use the fact that maxlength is `None` anywhere intentionally?

Reply:
good catch, foolish change on my part. Reverted, and the one type error in the same function extended to work when None.